public/app/features/alerting/unified/components/silences/SilenceGrafanaRuleDrawer.tsx

```diff
@@ -30,17 +27,10 @@ const SilenceGrafanaRuleDrawer = ({ rulerRule, onClose }: Props) => {
       size="md"
     >
       <Stack direction={'column'}>
-        <GrafanaAlertmanagerDeliveryWarning currentAlertmanager={selectedAlertmanager!} />
-
-        <div>
-          <AlertManagerPicker showOnlyReceivingGrafanaAlerts />
-          <Divider />
-        </div>
-
         <SilencesEditor
           ruleUid={uid}
           formValues={formValues}
-          alertManagerSourceName={selectedAlertmanager!}
+          alertManagerSourceName={GRAFANA_RULES_SOURCE_NAME}
```

Comment:
Now assuming that if you're creating a silence through the drawer, it'll be for Grafana AM